model.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, input_size, hidden_size, rnn_type=nn.LSTM, bidirectional=Fals
                             bidirectional=bidirectional, bias=False)
         self.num_directions = 2 if bidirectional else 1
 
+    def flatten_parameters(self):
+        self.rnn.flatten_parameters()
```

Comment:
I haven't ran this for a long time on multiple GPUs, but doesn't [this](https://github.com/SeanNaren/deepspeech.pytorch/blob/master/model.py#L66) solve the problem?

Reply:
Doesn't seem to, no. Even with the above change, I still get the warning, but less often.